### ISSUE TITLE
IE 9 and below ,I used 'alert()' and the placeholder no longer effective.

### ISSUE THREAD

**Issue by tdoly** [mathiasbynens/jquery-placeholder#178]
![ie9_before_click](https://f.cloud.github.com/assets/3241838/1780111/86243d08-6863-11e3-8630-80d8939bb121.png)
![ie9_after_click](https://f.cloud.github.com/assets/3241838/1780112/8644ef80-6863-11e3-8d4f-cc9b6fda3241.png)

I use placeholder on IE9. I check some input and use 'alert' prompt that the input is null.
1.when I click the button on the 'alert' ,the placeholder no longer effective in this page.
2.Don't input something,and check input.when I input the password in 'type=password',the password type is't effective.

this is demo:

<pre><code>
      &lt;!DOCTYPE html&gt;
      &lt;html lang="en"&gt;
        &lt;head&gt;
          &lt;meta charset="utf-8"&gt;
          &lt;title&gt;HTML5 placeholder jQuery Plugin Test&lt;/title&gt;
          &lt;script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"&gt;&lt;/script&gt;
          &lt;script src="jquery.placeholder.js"&gt;&lt;/script&gt;
          &lt;style&gt;
            body, input, textarea { font: 1em/1.4 Helvetica, Arial; }
            label code { cursor: pointer; float: left; width: 150px; }
            input { width: 14em; }
            textarea { height: 5em; width: 20em; }
            .placeholder { color: #aaa; }
            .note { border: 1px solid orange; padding: 1em; background: #ffffe0; }
            /* #p { background: lime; } */
        &lt;/style&gt;
        &lt;/head&gt;

        &lt;body&gt;
          &lt;h1&gt;HTML5 Placeholder jQuery Plugin Test&lt;/h1&gt;
          &lt;form&gt;
            &lt;p&gt;
              &lt;label&gt;
                &lt;code&gt;type=tel&lt;/code&gt; 
                &lt;input type="tel" name="tel" placeholder="please input telphone num"&gt;
              &lt;/label&gt;
            &lt;/p&gt;
            &lt;p&gt;
              &lt;label for="p"&gt;
                &lt;code&gt;type=password&lt;/code&gt; 
              &lt;/label&gt;
              &lt;input type="password" name="password" placeholder="please input password" id="p"&gt;
            &lt;/p&gt;
            &lt;p&gt;
              &lt;label for="p"&gt;
                &lt;code&gt;type=confirmpassword&lt;/code&gt; 
              &lt;/label&gt;
              &lt;/label&gt;&lt;input type="password" name="confirmpassword" placeholder="please input confirm password" id="p1"&gt;
            &lt;/p&gt;
            &lt;a href="javascript:void(0)" onclick="javascript:check()"&gt;check&lt;/a&gt;
          &lt;/form&gt;
          
          &lt;script&gt;
            $(function() {
              // Invoke the plugin
              $('input, textarea').placeholder();
            });

            function check() {
              var tel=$("input[name='tel']").val();
              var pwd=$("#p").val();
              var cpwd=$("#p1").val();
              if(tel == '') {
                alert("the telphone num is null");
              } else if(pwd == '') {
                alert("the password is null");
              } else if(cpwd == '') {
                alert("the confirm password is null");
              } else {
                alert("congratulations!");
              }
            }
          &lt;/script&gt;
        &lt;/body&gt;
      &lt;/html&gt;

</code></pre>
